packages/react-native-win32/src/Libraries/Utilities/Platform.win32.js

```diff
@@ -28,7 +28,22 @@ const Platform = {
     |},
   |} {
     if (this.__constants == null) {
-      this.__constants = NativePlatformConstantsWin.getConstants();
+      // Hack: We shouldn't need to null-check NativePlatformContants, but
+      // needed to remove the invariant it is non-null since react-native-win32
+      // hasn't picked up the changes we've made in 0.61 to add the module yet.
+      // This can be removed when we fix win32 NativePlatformConstantsWin.
+      if (NativePlatformConstantsWin) {
+        this.__constants = NativePlatformConstantsWin.getConstants();
+      } else {
+        this.__constants = {
+          isTesting: false,
+          reactNativeVersion: {
+            major: 0,
+            minor: 61,
```

Comment:
This will scream loudly if it gets out of date since I added the version check back. We should hopefully be using the Native module before we upgrade to 0.62, though I wouldn't be surprised if we're not.